src/components/carsList/index.js

```diff
@@ -1,12 +1,18 @@
 import React, { Component } from 'react'
 import CarCard from '../carCard'
 import { connect } from 'react-redux'
+import { Icon } from 'antd'
 
 class CarsList extends Component {
   render() {
     const { cars } = this.props
 
-    return cars.map((car, index) => <CarCard key={index} {...car} />)
+    return cars.map((car, index) =>
+      <div style={{ marginTop: 20, position: 'relative' }}>
```

Comment:
should we use https://ant.design/components/layout/

Reply:
I wanted to add some inline css